examples/enable/basic_draw.py

```diff
@@ -19,16 +21,18 @@ def _draw_mainlayer(self, gc, view_bounds=None, mode="default"):
             gc.fill_path()
 
 
-class MyFrame(DemoFrame):
+class Demo(HasTraits):
```

Comment:
This, and other similar changes, breaks the ability for the `pyglet` backend to run stand-alone examples (there is no TraitsUI support for pyglet).  Rather than replace `DemoFrame` in all the examples, instead replace the `DemoFrame` implementation here https://github.com/enthought/enable/blob/master/enable/example_support.py#L69 with a TraitsUI class (where `demo_main` just calls `configure_traits`) and refactor so that `create_window` is replaced by an abstract method that creates the top-level `component` and the pyglet `demo_main` is responsible for creating the `Window`.

Reply:
OTOH, I'm not sure how much we care about the pyglet backend.